docs/integrations/scalacheck.md

```diff
@@ -0,0 +1,142 @@
+---
+id: scalacheck
+title: ScalaCheck
+---
+
+MUnit supports writing property-based tests using
+[ScalaCheck](http://www.scalacheck.org/).
+
+## Getting Started
+
+ScalaCheck support is provided as a separate module. You can add it to your
+build via:
+
+```scala
+libraryDependencies += "org.scalameta" %% "munit-scalacheck" % "@VERSION@" % Test
+```
+
+You can then extend `ScalaCheckSuite` and write ScalaCheck property tests:
+
+```scala mdoc
+import munit.scalacheck.ScalaCheckSuite
+import org.scalacheck.Prop._
+
+class IntegerSuite extends ScalaCheckSuite {
+
+  property("addition is commutative") {
+    forAll { (n1: Int, n2: Int) =>
+      n1 + n2 == n2 + n1
+    }
+  }
+
+  property("0 is the identity of addition") {
+    forAll { (n: Int) =>
+      n + 0 == n
+    }
+  }
+
+}
+```
+
+> 👉 `property` is almost identical to `FunSuite`'s `test`, but it additionally
+> ensures that the test value is a ScalaCheck `Prop`.
+>
+> It supports all of `test`'s features, such as tagging and marking the property
+> as expected to fail:
+>
+> ```scala
+> property("my property".tag(WindowsOnly)) {
+>   forAll { (n: Int) => n * 0 == 0 }
+> }
+> ```
+>
+> ```scala
+> property("issue-123".fail) {
+>   forAll { (n: Int) => buggyFunction(n) }
+> }
+> ```
+
+## Configuring ScalaCheck
+
+You can override the `scalaCheckTestParameters` property of `ScalaCheckSuite` to
+customize how ScalaCheck checks the properties:
+
+```scala mdoc:reset
+import munit.scalacheck.ScalaCheckSuite
+import org.scalacheck.Prop._
+
+class IntegerSuite extends ScalaCheckSuite {
+
+  override def scalaCheckTestParameters =
+    super.scalaCheckTestParameters
+      .withMinSuccessfulTests(200)
+      .withMaxDiscardRatio(10)
+
+  property("addition is commutative") {
+    forAll { (n1: Int, n2: Int) =>
+      n1 + n2 == n2 + n1
+    }
+  }
+
+}
+```
+
+## Migrating from ScalaTest
+
+ScalaTest provides two styles for writing property-based tests, which are both
+front-ends to ScalaCheck.
+
+If you are using the "ScalaCheck style" API, i.e. the `Checkers` trait,
+switching to MUnit requires very minor and mechanical changes:
+
+```diff
+-import org.scalatest.prop.Checkers
+-import org.scalatest.FunSuite
++import munit.scalacheck.ScalaCheckSuite
```

Comment:
Reading the docs now makes we wonder if it's a good idea to call this package `scalacheck`. It might be better to put this in the `munit` toplevel package. If users do `import munit._` then they get an unwanted `scalacheck` symbol in scope. What do you think?

Reply:
I thought about it, and since scalacheck is prefixed by “org” I didn’t think it would be an issue; although I understand where you’re coming from.
I think it may make sense to either put into the toplevel package, or go the other way and do munit.integrations.scalacheck so that the intermediate name is less prone to clashes. I’m fine with either solutions

Reply:
I’m personally a fan of large packages. There’s low risk of namespace conflicts even if the scalacheck class lives in the munit package. I can do the move if you’re ok with it.

Reply:
Sure! :)

Reply:
Actually, my son woke up very early, so... done!
I've done in this PR so to merge the new docs and the change together